src/workspace-data/ImportStatus.js

```diff
@@ -78,11 +83,13 @@ function ImportStatusItem({ job: { targetWorkspace, jobId }, onDone }) {
 
     const errorNotify = () => notify('error', 'Error importing data.', { message });
 
-    if (!_.includes(status, ['Pending', 'Translating', 'ReadyForUpsert', 'Upserting'])) {
+    if (!_.includes(status, ['Pending', 'Translating', 'ReadyForUpsert', 'Upserting', 'RUNNING', 'CREATED', 'QUEUED'])) {
```

Comment:
I don't love co-mingling the enums here -- this probably deserves two separate branches of logic.

Reply:
I agree it's a bit ugly, but checking to determine which cloud it is and then having the exact same behavior is also a bit ugly.  Hm.